projects/croniter/project.yaml

```diff
@@ -5,6 +5,7 @@ language: python
 main_repo: https://github.com/kiorky/croniter
 sanitizers:
 - address
+- undefined
```

Comment:
You can leave this out since croniter doesn't compile native code -- just having `address` is enough